pkgs/by-name/i3/i3xrocks/package.nix

```
@@ -0,0 +1,48 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  autoconf,
+  automake,
+  pkg-config,
+  xcbutilxrm,
+  xorg,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "i3xrocks";
+  version = "1.3.6-1";
+
+  src = fetchFromGitHub {
+    owner = "regolith-linux";
+    repo = "i3xrocks";
+    rev = "v${version}";
+    hash = "sha256-qZWuYRwgSNVURxaZDzBU6yUcpIMUeGZg3HXlI8KzyX4=";
+  };
+
+  buildInputs = [
+    autoconf
+    automake
+    pkg-config
+    xcbutilxrm
+    xorg.xcbutil
+  ];
+
+  configurePhase = ''
+    ./autogen.sh
+    ./configure --prefix=$out
+  '';
```

Use `autorencofHook`
```suggestion
  nativeBuildInputs = [ autoreconfHook ];
```